diff --git a/src/networks/donkey/donkeyOptions.ml b/src/networks/donkey/donkeyOptions.ml
index 51a8cb6c..a324fba2 100644
--- a/src/networks/donkey/donkeyOptions.ml
+++ b/src/networks/donkey/donkeyOptions.ml
@@ -44,11 +44,10 @@ let client_timeout = define_expert_option donkey_section ["client_timeout"]
 
 let max_connected_servers = define_option donkey_section ["max_connected_servers"]
   "The number of servers you want to stay connected to, maximum allowable = 3"
-    int_option 3
+    int_option 9999
 
 let max_allowed_connected_servers () =
-  min (int_of_string (strings_of_option max_connected_servers).option_default)
-      !!max_connected_servers
+  9999
 
 let reliable_sources = define_option donkey_section ["reliable_sources"]
   "Should mldonkey try to detect sources responsible for corruption and ban them, currently disabled"
